word-search.py

```diff
@@ -400,7 +399,8 @@ def new_game():  #merged and renamed word_search() into new_game()
             else:
                 print("Invalid word. Try again.")
 
-def clear_screen(): #clear consoles
-    os.system('cls' if os.name == 'nt' else 'clear') 
+def clear_screen(): #clear console
```

Comment:
Performed review as requested. No conflicting issues found, working as expected 👍 Will merge to main